wolfcrypt/src/port/kcapi/kcapi_ecc.c

```diff
@@ -238,25 +278,27 @@ int KcapiEcc_Verify(ecc_key* key, const byte* hash, word32 hashLen, byte* sig,
     }
 
     if (ret == 0) {
-        sigHash = (unsigned char*)XMALLOC(sigLen + hashLen, key->heap,
-                                          DYNAMIC_TYPE_TMP_BUFFER);
-        if (sigHash == NULL) {
+        ret = posix_memalign((void*)&sigHash_aligned,
```

Comment:
Why not implement the same alignment check like the sign in the verify?

Reply:
We are passing both the sig and the hash together in a single buffer. There will always be a new buffer allocated.